src/order/OrderAPI.ts

```diff
@@ -94,6 +94,12 @@ export interface FilledOrder extends BasePlacedOrder {
   status: OrderStatus.DONE;
 }
 
+/** @see https://docs.pro.coinbase.com/#list-orders */
+export interface OrderListQueryParams extends Pagination {
+  product_id?: string;
+  status?: 'open' | 'pending' | 'active' | 'all';
```

Comment:
The Coinbase documentation states that multiple statuses can be requested. As such, the `status` property should accept either a single status string, or an array of status strings.